cmd/exporter.go

```diff
@@ -32,6 +33,7 @@ import (
 var (
 	address     = flag.String("address", "0.0.0.0:8888", "bind address")
 	metricsPath = flag.String("metrics-path", "/metrics", "metrics path")
+	enableGPU   = flag.Bool("enable-gpu", false, "whether enable gpu (need to have libnvidia-ml installed)")
```

Comment:
@metacosm the operator can decide to turn this option on if the node has [gpu labels](https://kubernetes.io/docs/tasks/manage-gpus/scheduling-gpus/)